src/ActivityLogServiceProvider.php

```diff
@@ -8,7 +8,10 @@ class ActivityLogServiceProvider extends ServiceProvider
 {
     public function register(): void
     {
-        //TODO GOVUK SERVICE PROVIDER
+        $this->mergeConfigFrom(
+            __DIR__ . '/activity-log.php',
+            'activity-log',
```

Comment:
The Spatie config is called `activitylog`. This could get very confusing. Find another name.